includes/class-wp-document-revisions-admin.php

```diff
@@ -1362,4 +1365,67 @@ public function make_private() {
 		$post = apply_filters( 'document_to_private', $post, $post_pre );
 
 	}
+
+
+	/**
+	 * Set up revisions on admin dashboard
+	 * @ since 3.0.1
+	 */
+	function setup_dashboard() {
+		wp_add_dashboard_widget('wpdr_dashboard', 
+		                        'Recently Revised Documents', 
+		                        array( &$this, 'dashboard_display' ) );
+	}
+
+
+	/**
+	 * Callback to display documents on admin dashboard
+	 * @ since 3.0.1
+	 */
+	function dashboard_display () {
+
+		global $wpdr;
+		if ( ! $wpdr ) {
+			$wpdr = Document_Revisions::$instance;
+		}
+
+		$query = array(
+			'orderby'     => 'modified',
+			'order'       => 'DESC',
+			'numberposts' => 5,
+		);
+
+		$documents = $wpdr->get_documents( $query );
+
+		// no documents, don't bother
+		if ( ! $documents ) {
+			return;
+		}
+
+		// @codingStandardsIgnoreLine WordPress.XSS.EscapeOutput.OutputNotEscaped
```

Comment:
What can't be escaped here?

Reply:
This could certainly be a misunderstanding on my part. I copied the code from the widget code. This has
```
	// @codingStandardsIgnoreLine WordPress.XSS.EscapeOutput.OutputNotEscaped
	echo $args['before_widget'] . $args['before_title'] . esc_html( apply_filters( 'widget_title', $instance['title'] ) ) . $args['after_title'] . '<ul>';
```
Clearly a lot of these elements are not needed here so it reduces to: 
```
        // @codingStandardsIgnoreLine WordPress.XSS.EscapeOutput.OutputNotEscaped
        echo '<ul>';
```
I have tried to look up the documentation about (not) escaping html text but Google seems not to be my friend. So whether I need to mark the line as ignored or not given it is fixed text is unclear to me. I thought it safest to leave it there.

I'm happy to remove the comment here (and for the end list tag below) if you request.

Neil